src/lib/components/drip-list-card/drip-list-card.svelte

```diff
@@ -226,6 +228,7 @@
   href={dripListUrl}
   target={openInNewTab ? '_blank' : undefined}
   class="drip-list-card rounded-drip-lg overflow-hidden shadow-low group"
+  class:hidden-list={isHidden}
```

Comment:
You have `hidden-list` and a `hiddenList` class above. How about keeping it kebab case?

Reply:
👍